test/unit/compiler.spec.js

```diff
@@ -30,4 +30,31 @@ describe("Peggy compiler", () => {
       output: "INVALID OUTPUT TYPE",
     })).to.throw("Invalid output format: INVALID OUTPUT TYPE.");
   });
+
+  it("generates inline sourceMappingURL", () => {
+    const ast = parser.parse("foo='1'");
+    // Ensure there is an expect even if we don't run the good tests below.
```

Comment:
```suggestion
    // Ensure there is an object even if we don't run the good tests below.
```
Misprint, I guess

Reply:
eh, i'll just remove the comment.  what i'm aiming for is that if you're running in an old browser, you don't see an error, and the `it` contains at least one `expect`.